### ISSUE TITLE
Support Public vs Private Instruments

### ISSUE THREAD

**Issue by gdevenyi** [DouglasNeuroInformatics/OpenDataCapture#630]
Since we're going to be integrating "public" instruments, that is, ones implemented from the literature and standardized across groups, we would like a way to seperate the private/custom from the public. Its possible tags could be used for this. It doesn't need to be "security" level hidden, just convenience hidden.

**Comment by joshunrau** [comment]
I was going to add some kind of functionalities for group managers to toggle visibility of instruments. Do you think that would cover this?

**Comment by gdevenyi** [comment]
Yes, it can, this is mostly a hint for people to "pay attention to these, these are useful" vs, "unless this is from your group you don't care"

**Comment by gdevenyi** [comment]
Triaging upstream: group-level accessibleInstrumentIds appears to partially address access control. Is there still a separate need for explicit public/private instrument tagging, or can this be closed?
